content/en/tracing/trace_collection/tracing_naming_convention/_index.md

```diff
@@ -14,112 +14,63 @@ further_reading:
-| `rpc.service` | `string` | The name of the service being called. |
-| `rpc.method`  | `string` | The name of the method being called.  |
-
-### Errors
-The following span tags can be used to describe errors associated with spans:
-
-| **Name**    | **Type** | **Description**                                                  |
-|-----------------|----------|------------------------------------------------------------------|
-| `error.type` | `string` | The error type or kind (or code in some cases).                  |
-| `error.message`    | `string` | A concise, human-readable, one-line message explaining the event. |
-| `error.stack`   | `string` | The stack trace or the complementary information about the error. |
+Each span consists of [span tags][2] to provide additional information on the unit of work happening in the system. Naming conventions describe the name and content that can be used in span events.
+
+<div class="alert alert-info">To find a comprehensive list of all span tags, reserved attributes, and naming conventions, see <a href="/standard-attributes/?product=apm">Default Standard Attributes.</a></div>
+
+## Span tag naming conventions
+
+There are a variety of span tags to describe work happening in the system. For example, there are span tags to describe the following domains:
+
+- **Reserved**: Attributes that are always present on all spans.
+- **Core**: Instrumentation used and the kind of operation.
+- **Network communications**: Work units corresponding to network communications.
+- **HTTP requests**: HTTP client and server spans.
+- **Database**: Database spans.
+- **Message queue**: Messaging system spans.
+- **Remote procedure calls**: Spans corresponding to remote procedure calls such as RMI or gRPC.
+- **Errors**: Errors associated with spans.
+
+For more information, see [Default Standard Attributes][6].
+
+## Span tags and span attributes
+
+Span tags and span attributes are similar but distinct concepts:
+
+- [Span tags](#span-tags) are the context around the span.
+- [Span attributes](#span-attributes) are the content of the span.
+
+### Span tags
+
+Span tags are the context around the span. Some examples include:
+
+- **Host tags**: `hostname`, `availability-zone`, `cluster-name`
+- **Container tags**: `container_name`, k`ube_deployment`, `pod_name`
+
+Tags are usually enriched from other data sources like host tags, container tags, or service catalog tags. These tags are added to the span to describe the context. For example, tags might describe the properties of the host and the container the span is coming from, or the properties of the services the span is emitted from.
+
+One span can have multiple tags with the same key. For example, consider a `team` tag. You might have one `team` tag that describes the team owning the infrastructure layer (host tag), and another `team` tag describing the team owning the service (service catalog tag).
```

Comment:
I would get rid of this part. The multiple values with the same key does not work everywhere and can be misleading